pipelines/placesapi/README.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Places API
+description: dlt pipeline for Places API
+keywords: [places api, places pipeline]
+---
+
+# Places API
+
+The pipleline uses [Google Places API](https://developers.google.com/maps/documentation/places/web-service/overview) to search information about different locations, and loads data into [Google BigQuery](https://dlthub.com/docs/destinations/bigquery).
+
+
+## Initialize the pipeline
+
+Initialize the pipeline with the following command:
+
+```
+dlt init placesapi bigquery
+```
+
+Running this command will create a directory with the following structure:
+
+```bash
+├── .dlt
+│   ├── .pipelines
+│   ├── config.toml
+│   └── secrets.toml
+├── placesapi
+│   └── __pycache__
```

Comment:
When everything is set up correctly `__pycache__` should not be here. Let me know if it's not ignored by gitignore in your case.